src/graphql/types/object/transaction.ts

```diff
@@ -0,0 +1,75 @@
+import dedent from "dedent"
+import { connectionDefinitions } from "graphql-relay"
+
+import { GT } from "@graphql/index"
+
+import Memo from "../scalar/memo"
+import Price from "./price"
+import SatAmount from "../scalar/sat-amount"
+import InitiationVia from "../abstract/initiation-via"
+import SettlementVia from "../abstract/settlement-via"
+import Timestamp from "../scalar/timestamp"
+import TxDirection, { txDirectionValues } from "../scalar/tx-direction"
+import TxStatus from "../scalar/tx-status"
+
+const Transaction = new GT.Object({
+  name: "Transaction",
+  description: dedent`Give details about an individual transaction.
+  Galoy have a smart routing system which is automatically
+  settling intraledger when both the payer and payee use the same wallet
+  therefore it's possible the transactions is being initiated onchain
+  or with lightning but settled intraledger.`,
+  fields: () => ({
+    id: {
+      type: GT.NonNullID,
+    },
+    initiationVia: {
+      type: GT.NonNull(InitiationVia),
+      description: "From which protocol the payment has been initiated.",
+    },
+    settlementVia: {
+      type: GT.NonNull(SettlementVia),
+      description: "To which protocol the payment has settled on.",
+    },
+    settlementAmount: {
+      type: GT.NonNull(SatAmount),
+      description: "Amount of sats paid by the user.",
```

Comment:
this is inclusive fees? maybe should be added to the description?

Reply:
yes, this amount includes fee

![image](https://user-images.githubusercontent.com/2079600/143912310-0d2e714c-f179-47d9-b855-66dfd3cb3076.png)